index.js

```diff
@@ -59,10 +65,10 @@ module.exports.serverless = appFn => {
 
     // Bail for null body
     if (!event.body) {
-      return context.done(null, {
```

Comment:
This is the only part Im not sure of, can you clarify why the callback was removed?

Reply:
See #18, apparently it is no longer in use. It was just missed in that pr so I doubled up some work